integration/docker-compose.spdk.yml

```diff
@@ -30,6 +29,27 @@ services:
             ./rpc.py nvmf_subsystem_add_listener nqn.2016-06.io.spdk:cnode1 -t tcp -a 10.129.129.4 -s 4420  && \
             ./rpc.py nvmf_subsystem_add_ns nqn.2016-06.io.spdk:cnode1 Malloc0 -n 1  && \
             ./rpc.py nvmf_subsystem_add_ns nqn.2016-06.io.spdk:cnode1 Malloc1 -n 2  && \
+            wait'
+    healthcheck:
+      test: ["CMD-SHELL", "python3 /usr/libexec/spdk/scripts/rpc.py spdk_get_version || exit 1"]
+      interval: 6s
+      retries: 5
+      start_period: 20s
+      timeout: 10s
+
+  spdk-target-web:
+    image: ghcr.io/opiproject/opi-spdk:main
+    volumes_from:
+      - spdk-target:rw
+    ports:
+      - "9004:9009"
+    networks:
+      high-speed-external:
+        ipv4_address: 10.129.129.8
```

Comment:
Update the diagram with this new IP address and component.

Reply:
will do